### ISSUE TITLE
Package listing without Package.swift: azure-sdk-for-ios

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1673]
The nightly is correctly [flagging the following package](https://github.com/SwiftPackageIndex/PackageList/runs/5809512914?check_suite_focus=true) without a Package.swift:

```
Warning: invalid package: https://github.com/Azure/azure-sdk-for-ios.git: The file “Package.swift” couldn’t be opened.
```

https://swiftpackageindex.com/Azure/azure-sdk-for-ios

The package page displays ok-ish, although there are obviously no successful builds.

The README states

> To install the Azure client libraries for iOS, we recommend you use [Swift Package Manager](https://swiftpackageindex.com/search?query=azure#add-a-package-dependency-with-swift-package-manager). As an alternative, you may also integrate the libraries using [CocoaPods](https://swiftpackageindex.com/search?query=azure#integrate-the-client-libraries-with-cocoapods).

but I don't see how this would actually work.

Ok, I see now there's a `Package.swift` for the latest tag. So that seems to be the same mechanism we saw with another package (or was it this one?) recently.

I don't think there's no action required here, although it's notable that this package couldn't really be added to the index in the first place right now. I guess it had a Package.swift on the default branch in the past 🤷.

**Comment by finestructure** [comment]
Just going to close this again but at least we have a record of the observation.

**Comment by daveverwer** [comment]
Yes, it was an Amazon package where we first saw this. I don't remember the name, but I remember creating an issue, and they said they were happy with how it worked and were not planning to change it.

I just went looking for details, and I wasn't quite right. It's the [aws-sdk-swift](https://github.com/awslabs/aws-sdk-swift) package. They have a [Package.swift](https://github.com/awslabs/aws-sdk-swift/blob/main/Package.swift), but it doesn't compile on anything apart from the tagged releases. The issue I made is [still open](https://github.com/awslabs/aws-sdk-swift/issues/505) but from the response, I don't think they're planning to change things.

I had no interest in coping with the situation when it was one package. I feel the same with two, but we may have to provide a fallback if this becomes a common problem. If not for the validator, for the package index itself where it doesn't show the default branch, just the latest tagged release.

That's going to be a lot of work, though. We assume that a package will have a default branch *everywhere*.
